src/celeritas/global/ActionRegistry.cc

```diff
@@ -49,7 +49,7 @@ void ActionRegistry::insert_const_impl(SPConstAction&& action)
  */
 void ActionRegistry::insert_impl(SPConstAction&& action)
 {
-    auto label = action->label();
+    auto label = std::string{action->label()};
```

Comment:
same here,`string_view` as the key for the map, then `labels_.emplace_back(label)`